### DISCUSSION TITLE
PS model in Martini3 force field

### DISCUSSION THREAD

**Discussion by rswf** [marrink-lab/polyply_1.0#379] [Martini3]
Hello! I am working on a polymer simulation based on Martini3 force field. I noticed that there are Martini3 models for polystyrene in the polyply project. Comparison of the bonding parameters shows that the model is same with the Martini2 force field based model proposed in the literature [(Coarse-graining polymers with the MARTINI force-field: polystyrene as a benchmark case).](https://doi.org/10.1039/C0SM00481B) With the bonded parameters unchanged, the bead type becomes that in Martini3. May I ask if there is a reference to support such a change? Is the changed model validated?

**Comment by fgrunewald** [comment]
@rswf you are correct. This approximation is okish but we have new updated parameters already. Hopefully, next week we can upload them to the library and make them available. A benchmark of the new parameters is avilbale [here](https://research.rug.nl/en/publications/material-design-using-martini-accelerating-discovery-through-coar).

**Reply by rswf** [reply]
Thanks for your reply! This is indeed great news. And I think this paper is very meaningful for my research, thanks!

**Reply by rswf** [reply]
Hi @fgrunewald  , I am performing simulations of the swelling of block copolymers with PS as the main component under Martini3 force field and I have encountered some problems.

Firstly, thanks for the suggestion provided in your paper to use SN6r as ethanol, which solves the problem of the immiscibility of ethanol and some solvents with benzene rings in their structure (e.g. toluene). But this introduces a new problem, the PS-ethanol interaction seems to be too strong, and the block copolymers rapidly micellise in ethanol (1 μs), which does not correspond well to the results in the experiments.

I constructed PS based on ethylbenzene from the Martini3 small molecule database, and I am not sure if there is a problem with the model parameters. 
[M3-PS100-itp.txt](https://github.com/user-attachments/files/17304424/M3-PS100-itp.txt)
So I would like to request your PS model as a reference for testing, I would appreciate if you agree!

Looking forward to your reply.
